package.json

```diff
@@ -30,7 +30,7 @@
   "license": "MIT",
   "devDependencies": {
     "@types/jest": "^24.0.1",
-    "bitmovin-player": "^8.165.0",
+    "bitmovin-player": "^8.191.0",
```

Comment:
Do you mind updating the player also in the sample page since it's using a very old one?
https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/blob/develop/example/index.html#L36

Reply:
Not needed anymore.